### PR TITLE
style: update color and styles in metrics for Feedback Task

### LINKED ISSUE #2908: UI: final fine-tuning for Feedback Task
# Update styles

- [x]  Remove record counter in footer (Leire)
- [x]  Change form title: Submit your feedback (Leire)
- [x]  Add style to info question tooltip (leire)
- [x]  Metrics style
- [x]  Correct spaces between form and record
- [x]  Add style to guidelines
- [x]  Correct colour of the form stroke:  #fd8f5e → #bb720a
- [x]  Comment part of the sidebar (Leire)
    - [x]  Comment hand labeling part (title+icon+tooltip)
    - [x]  Comment “metrics” title and “refresh” title
    - [x]  Maintain only the 2 icons + tooltip
- [x]  Add Link to user guidelines. Read the annotation guidelines (Leire) Add icon

### PR DESCRIPTION
**Pull request by leiyre**
# Description

This PR includes some styling to update the colour in the progress sidebar in Feedback Task

Closes #2908

**Type of change**

(Please delete options that are not relevant. Remember to title the PR according to the type of change)

- [x] Styles

**Checklist**

- [x] I have merged the original branch into my forked branch
- [ ] I added relevant documentation
- [x] follows the style guidelines of this project
- [x] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue [hunk 1]
```diff
@@ -90,6 +103,15 @@ export default {
       return this.totalResponded / this.progressTotal;
     },
   },
+  methods: {
+    capitalize(text) {
+      const baseText = text.toLowerCase();
+      return `${baseText.charAt(0).toUpperCase()}${baseText.slice(1)}`;
+    },
```
Comment (keithCuniah) [inline frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue]: this kind of global function should be external of the component

Comment (leiyre) [reply frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue]: yes, I have replaced it with a existed filter

File: frontend/plugins/filters.js [hunk 2]
```diff
@@ -22,8 +22,9 @@ Vue.filter("percent", function (value, min, max) {
 });
 
 Vue.filter("capitalize", function (value) {
+  const baseValue = value.toLowerCase();
   const capitalize = ([firstLetter, ...restOfWord]) =>
     firstLetter.toUpperCase() + restOfWord.join("");
 
-  return capitalize(value);
+  return capitalize(baseValue);
```
Comment (keithCuniah) [inline frontend/plugins/filters.js]: choose a more explicit name value than baseValue => something like 
textInLowerCase

File: frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue [hunk 3]
```diff
@@ -26,36 +26,28 @@
...
+          <span
+            class="color-bullet"
+            :style="{ backgroundColor: status.color }"
+          ></span>
+          <label class="metrics__list__name">{{
+            status.name | capitalize
+          }}</label>
+          <span class="metrics__list__counter">{{
+            status.progress | formatNumber
+          }}</span>
```
Comment (keithCuniah) [inline frontend/components/commons/sidebar-feedback-task/SidebarFeedbackTaskProgress.vue]: be carefull of null exception, if status.name/progress is null, then there will be an error

**Review by keithCuniah — APPROVED** [review]

**Pull request merged**